commands/cluster-nodes.md

```diff
@@ -17,57 +17,90 @@ It is also used by `redis-cli` in order to manage a cluster.
+292f8b365bb7edb5e285caf0b7e6ddc7265d2f4f 127.0.0.1:30003@31003,,shard-id=fdb36c73e72dd027bc19811b7c219ef6e55c550e master - 0 1426238318243 3 connected 10923-16383
+6ec23923021cf3ffec47632106199cb7f496ce01 127.0.0.1:30005@31005,,shard-id=114f6674a35b84949fe567f5dfd41415ee776261 slave 67ed2db8d677e59ec4a4cefb06858cf2a1a89fa1 0 1426238316232 5 connected
+824fe116063bc5fcf9f4ffd895bc17aee7731ac3 127.0.0.1:30006@31006,,shard-id=fdb36c73e72dd027bc19811b7c219ef6e55c550e slave 292f8b365bb7edb5e285caf0b7e6ddc7265d2f4f 0 1426238317741 6 connected
+e7d1eecce10fd6bb5eb35b9f99a514335d9ba9ca 127.0.0.1:30001@31001,,shard-id=69bc080733d1355567173199cff4a6a039a2f024 myself,master - 0 0 1 connected 0-5460
 ```
 
 Each line is composed of the following fields:
 
 ```
-<id> <ip:port@cport> <flags> <master> <ping-sent> <pong-recv> <config-epoch> <link-state> <slot> <slot> ... <slot>
+<id> <ip:port@cport[,hostname[,auxiliary_field=value]*]> <flags> <master> <ping-sent> <pong-recv> <config-epoch> <link-state> <slot> <slot> ... <slot>
 ```
 
 The meaning of each filed is the following:
 
-1. `id`: The node ID, a 40 characters random string generated when a node is created and never changed again (unless `CLUSTER RESET HARD` is used).
-2. `ip:port@cport`: The node address where clients should contact the node to run queries.
-3. `flags`: A list of comma separated flags: `myself`, `master`, `slave`, `fail?`, `fail`, `handshake`, `noaddr`, `nofailover`, `noflags`. Flags are explained in detail in the next section.
-4. `master`: If the node is a replica, and the master is known, the master node ID, otherwise the "-" character.
-5. `ping-sent`: Milliseconds unix time at which the currently active ping was sent, or zero if there are no pending pings.
-6. `pong-recv`: Milliseconds unix time the last pong was received.
-7. `config-epoch`: The configuration epoch (or version) of the current node (or of the current master if the node is a replica). Each time there is a failover, a new, unique, monotonically increasing configuration epoch is created. If multiple nodes claim to serve the same hash slots, the one with higher configuration epoch wins.
-8. `link-state`: The state of the link used for the node-to-node cluster bus. We use this link to communicate with the node. Can be `connected` or `disconnected`.
-9. `slot`: A hash slot number or range. Starting from argument number 9, but there may be up to 16384 entries in total (limit never reached). This is the list of hash slots served by this node. If the entry is just a number, is parsed as such. If it is a range, it is in the form `start-end`, and means that the node is responsible for all the hash slots from `start` to `end` including the start and end values.
-
-Meaning of the flags (field number 3):
+1. `id`: The node ID, a 40-character globally unique string generated when a node
+   is created and never changed again (unless `CLUSTER RESET HARD` is used).
```

Comment:
Leaving everything on a single line does a better job of preserving git history. It also makes it much easier to review changes since it doesn't flag the entire line as `new`.

Reply:
Ah, I thought we had a different preference for the documentation. Will revert next

Reply:
See https://github.com/redis/redis-doc/#styling-guidelines